<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>815</width>
    <height>467</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <underline>false</underline>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="font">
    <font>
     <underline>false</underline>
    </font>
   </property>
   <widget class="QTabWidget" name="mainTab">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>791</width>
      <height>420</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="thermographyTab">
     <property name="styleSheet">
      <string notr="true">font: 13pt &quot;Piboto Condensed&quot;;
</string>
     </property>
     <attribute name="title">
      <string>Thermal Camera</string>
     </attribute>
     <widget class="QLabel" name="lbl_image">
      <property name="geometry">
       <rect>
        <x>105</x>
        <y>35</y>
        <width>320</width>
        <height>240</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="imgBack">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>791</width>
        <height>391</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QComboBox" name="cameraBox">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>80</y>
        <width>220</width>
        <height>31</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0,0,0);
font: italic 12pt &quot;DejaVu Math TeX Gyre&quot;;
</string>
      </property>
      <item>
       <property name="text">
        <string>Choose Camera</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>SeekCompactPro</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ThermalExpertQ1Pro</string>
       </property>
      </item>
     </widget>
     <widget class="QGroupBox" name="groupBoxCamera">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>10</y>
        <width>251</width>
        <height>361</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Source Code Pro</family>
        <pointsize>20</pointsize>
        <weight>9</weight>
        <italic>false</italic>
        <bold>false</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 20pt &quot;Source Code Pro&quot;;</string>
      </property>
      <property name="title">
       <string>Camera Options</string>
      </property>
      <widget class="QPushButton" name="ACQButton">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>270</y>
         <width>71</width>
         <height>71</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(0,0,0, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="saveButton">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>270</y>
         <width>71</width>
         <height>71</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(0,0,0, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="startButton">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>150</y>
         <width>71</width>
         <height>71</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Source Code Pro</family>
         <pointsize>20</pointsize>
         <weight>9</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(0,0,0, 0);


</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="LoadButton">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>150</y>
         <width>71</width>
         <height>71</height>
        </rect>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Source Code Pro</family>
         <pointsize>20</pointsize>
         <weight>9</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(0,0,0, 0);

</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="Line" name="line">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>-10</y>
         <width>3</width>
         <height>380</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(136, 138, 133);</string>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </widget>
     <zorder>imgBack</zorder>
     <zorder>groupBoxCamera</zorder>
     <zorder>lbl_image</zorder>
     <zorder>cameraBox</zorder>
    </widget>
    <widget class="QWidget" name="configurationTab">
     <property name="styleSheet">
      <string notr="true">font: 25 13pt &quot;Piboto Condensed&quot;;</string>
     </property>
     <attribute name="title">
      <string>Configuration</string>
     </attribute>
     <widget class="QLabel" name="lbl_image2">
      <property name="geometry">
       <rect>
        <x>105</x>
        <y>35</y>
        <width>320</width>
        <height>240</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="imgBack_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>791</width>
        <height>391</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QComboBox" name="filterBox">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>70</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0,0,0);
font: italic 12pt &quot;DejaVu Math TeX Gyre&quot;;</string>
      </property>
      <item>
       <property name="text">
        <string>Choose Filter</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Gaussian Filter</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Homogeneous Blur</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Median Blur</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Bilateral Blur</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="kernelBox">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>120</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0,0,0);
font: italic 12pt &quot;DejaVu Math TeX Gyre&quot;;</string>
      </property>
      <item>
       <property name="text">
        <string>Select Kernel Size</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6</string>
       </property>
      </item>
     </widget>
     <widget class="QGroupBox" name="groupBoxConf">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>10</y>
        <width>251</width>
        <height>361</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Source Code Pro</family>
        <pointsize>20</pointsize>
        <weight>9</weight>
        <italic>false</italic>
        <bold>false</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 20pt &quot;Source Code Pro&quot;;</string>
      </property>
      <property name="title">
       <string>Config. Options</string>
      </property>
      <widget class="QPushButton" name="deleteButton">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>270</y>
         <width>71</width>
         <height>71</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(0,0,0, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="save2Button">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>170</y>
         <width>71</width>
         <height>71</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(0,0,0, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="undoButton">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>170</y>
         <width>71</width>
         <height>71</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(0,0,0, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="txtButton">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>270</y>
         <width>71</width>
         <height>71</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(0,0,0, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="Line" name="line_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>-10</y>
         <width>3</width>
         <height>380</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(136, 138, 133);</string>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </widget>
     <zorder>imgBack_2</zorder>
     <zorder>groupBoxConf</zorder>
     <zorder>lbl_image2</zorder>
     <zorder>filterBox</zorder>
     <zorder>kernelBox</zorder>
    </widget>
   </widget>
   <widget class="QTextBrowser" name="StatusBrowser">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>491</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
